### PR TITLE
1. Python SDK page updated.

### LINKED ISSUE #1430: Obsolete Python SDK content
Python SDK page is now obsolete, it meeds to be updated.

### PR DESCRIPTION
**Pull request by asladeofgreen**
### What does this PR fix/introduce?

Obsolete Python SDK page content.

Closes #1430

### Additional context

Simplified page by pointing directly to dedicated How To pages within SDK repo itself.

### Checklist
(Delete any that aren't relevant)

- [x] Docs are successfully building - `yarn install && yarn run build`.
- [x] All external links have been verified with `yarn run check:externals`.
- [x] My changes follow the [Casper docs style guidelines](https://docs.casper.network/resources/contribute-to-docs/).

### Reviewers
@ipopescu

### CODE REVIEW

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 1]
```diff
@@ -1,236 +1,53 @@
...
+pip3 install pycspr
 ```
 
-### Staking
-
-This example shows you how to define and stake funds with a validator.
+## How To's
 
-```python
+The following set of How To's cover the full SDK feature set & are designed to be run against a [CCTL](https://github.com/casper-network/cctl) network.
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:13]:
```suggestion
The following set of How To's cover the full SDK feature set and are designed to be run against a [CCTL](https://github.com/casper-network/cctl) network.
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 2]
```diff
@@ -1,236 +1,53 @@
...
-    import pathlib
+### Cryptography
 
-    import pycspr
-    from pycspr.client import NodeClient
-    from pycspr.client import NodeConnectionInfo
-    from pycspr.crypto import KeyAlgorithm
-    from pycspr.types import Deploy
-    from pycspr.types import PrivateKey
+* [How To: Apply a checksum ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_apply_a_checksum.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:17]:
```suggestion
* [How To: Apply a checksum](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_apply_a_checksum.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 3]
```diff
@@ -1,236 +1,53 @@
...
 
-    # path to cp1 secret key - defaults to NCTL user 1.
-    path_to_validator_secret_key = pathlib.Path(os.getenv("NCTL")) / "assets" / "net-1" / "users" / "user-1" / "secret_key.pem"
+* [How To: Create Key Pairs ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_create_key_pairs.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:19]:
```suggestion
* [How To: Create Key Pairs](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_create_key_pairs.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 4]
```diff
@@ -1,236 +1,53 @@
...
 
-    # type of cp1 secret key - defaults to ED25519.
-    type_of_validator_secret_key = KeyAlgorithm.ED25519.name
+* [How To: Hash data ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_hash_data.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:21]:
```suggestion
* [How To: Hash data](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_hash_data.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 5]
```diff
@@ -1,236 +1,53 @@
...
 
-    # path to session code wasm binary - defaults to NCTL bin/eco/add_bid.wasm.
-    path_to_wasm = pathlib.Path(os.getenv("NCTL")) / "assets" / "net-1" / "bin" / "auction" / "add_bid.wasm"
+* [How To: Sign data ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_sign_data.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:23]:
```suggestion
* [How To: Sign data](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/crypto/how_to_sign_data.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 6]
```diff
@@ -1,236 +1,53 @@
...
 
-    # amount to stake, i.e. bond, into the network.
-    amount = int(2.5e9)
+### Deploys
 
-    # amount to charge delegators for service provision.
-    delegation_rate = 2
+* [How To: Transfer funds between 2 accounts ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_transfer.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:27]:
```suggestion
* [How To: Transfer funds between 2 accounts](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_transfer.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 7]
```diff
@@ -1,236 +1,53 @@
...
 
-    # name of target chain - defaults to NCTL chain.
-    chain_name = "casper-net-1"
+* [How To: Delegate funds to a validator ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_delegate.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:29]:
```suggestion
* [How To: Delegate funds to a validator](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_delegate.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 8]
```diff
@@ -1,236 +1,53 @@
...
 
-    # host address of target node - defaults to NCTL node 1.
-    node_host = "localhost"
+* [How To: Undelegate funds from a validator ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_undelegate.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:31]:
```suggestion
* [How To: Undelegate funds from a validator](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_undelegate.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 9]
```diff
@@ -1,236 +1,53 @@
...
 
-    # Node API JSON-RPC port - defaults to 11101 @ NCTL node 1.
-    node_port_rpc = 11101
+* [How To: Stake funds as a validator ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_stake.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:33]:
```suggestion
* [How To: Stake funds as a validator](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_stake.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 10]
```diff
@@ -1,236 +1,53 @@
...
 
-    def _main(node_host, node_port_rpc, path_to_validator_secret_key, type_of_validator_secret_key, chain_name, amount, delegation_rate, path_to_wasm):
-        """Main entry point.
-        :param args: Parsed command line arguments.
-        """
-        # Set node client.
-        client: NodeClient = _get_client(node_host, node_port_rpc)
+* [How To: Unstake funds as a validator ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_unstake.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:35]:
```suggestion
* [How To: Unstake funds as a validator](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/deploys/how_to_unstake.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 11]
```diff
@@ -1,236 +1,53 @@
...
-        # Set validator key.
-        validator: PrivateKey = pycspr.parse_private_key(
-            path_to_validator_secret_key,
-            type_of_validator_secret_key,
-            )
+### Smart Contracts
 
-        # Set deploy.
-        deploy: Deploy = _get_deploy(validator, chain_name, amount, delegation_rate, path_to_wasm)
+* [How To: Install a smart contract ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/smart_contracts/how_to_install.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:39]:
```suggestion
* [How To: Install a smart contract](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/smart_contracts/how_to_install.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 12]
```diff
@@ -1,236 +1,53 @@
...
 
-        # Approve deploy.
-        deploy.approve(validator)
+* [How To: Invoke a smart contract ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/smart_contracts/how_to_invoke.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:41]:
```suggestion
* [How To: Invoke a smart contract](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/smart_contracts/how_to_invoke.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 13]
```diff
@@ -1,236 +1,53 @@
...
 
-        # Dispatch deploy to a node.
-        client.deploys.send(deploy)
+* [How To: Query a smart contract ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/smart_contracts/how_to_query.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:43]:
```suggestion
* [How To: Query a smart contract](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/smart_contracts/how_to_query.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 14]
```diff
@@ -1,236 +1,53 @@
...
 
-        print(f"Deploy dispatched to node [{node_host}]: {deploy.hash.hex()}")
+### Node APIs
 
+* [How To: Use REST API ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_rest_client.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:47]:
```suggestion
* [How To: Use REST API](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_rest_client.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 15]
```diff
@@ -1,236 +1,53 @@
...
 
-    def _get_client(node_host, node_port_rpc) -> NodeClient:
-        """Returns a pycspr client instance.
-        """
-        return NodeClient(NodeConnectionInfo(
-            host = node_host,
-            port_rpc = node_port_rpc,
-        ))
+* [How To: Use RPC API ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_rpc_client.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:49]:
```suggestion
* [How To: Use RPC API](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_rpc_client.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 16]
```diff
@@ -1,236 +1,53 @@
...
 
+* [How To: Use Speculative RPC API ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_speculative_rpc_client.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:51]:
```suggestion
* [How To: Use Speculative RPC API](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_speculative_rpc_client.py)
```

File: source/docs/casper/developers/dapps/sdk/python-sdk.md [hunk 17]
```diff
@@ -1,236 +1,53 @@
...
-            )
-
-        return deploy
-
-
-    # Entry point.
-    if __name__ == '__main__':
-        _main(node_host, node_port_rpc, path_to_validator_secret_key, type_of_validator_secret_key, chain_name, amount, delegation_rate, path_to_wasm)
-```
+* [How To: Use SSE API ?](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_sse_client.py)
```
Comment (ipopescu) [inline source/docs/casper/developers/dapps/sdk/python-sdk.md:53]:
```suggestion
* [How To: Use SSE API](https://github.com/casper-network/casper-python-sdk/blob/main/how_tos/node_apis/how_to_use_sse_client.py)
```

**Review by ipopescu — APPROVED** [review]
LGTM! I left a few minor nits. Removing the question marks seems cleaner.

**Comment by ipopescu** [comment]
The build will be fixed with this PR:
 https://github.com/casper-network/docs/pull/1429/

**Comment by asladeofgreen** [comment]
@ipopescu Incorporated your nits !

**Comment by ipopescu** [comment]
Great, then I will merge it.

**Pull request merged**